test/test_utils.h

```diff
@@ -40,11 +40,12 @@ class TestErrors : public Errors {
   std::vector<ErrorList> errors;
 
   void Clear();
+  bool HasError() const override;
 
  protected:
-  void HandlePushContext(Location loc, string_view desc);
-  void HandlePopContext();
-  void HandleOnError(Location loc, string_view message);
```

Comment:
Do we not have warning that requires `override` ?

Reply:
I guess not, seems that `-Wsuggest-override` will do it though.